app/lib/actions/katello/organization/simple_content_access/toggle.rb

```diff
@@ -0,0 +1,38 @@
+module Actions
+  module Katello
+    module Organization
+      module SimpleContentAccess
+        class Toggle < Actions::AbstractAsyncTask
+          middleware.use Actions::Middleware::PropagateCandlepinErrors
+
+          SIMPLE_CONTENT_ACCESS_DISABLED_VALUE = "entitlement".freeze
+          SIMPLE_CONTENT_ACCESS_ENABLED_VALUE = "org_environment".freeze
+
+          attr_reader :organization
+
+          def plan(organization_id)
+            @organization = ::Organization.find(organization_id)
+
+            fail N_('Not allowed to toggle Simple content access') unless consumer
```

Comment:
To get the upstream consumer there is also the option of `Katello::Resources::Candlepin::UpstreamCandlepinResource.upstream_consumer` which will raise an error if there is no manifest imported like we have in other areas - so we wouldn't need a new message here